tests/performance_evaluation.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
Make sure you have a good reason to add any file you do. Why are we adding this here?

Reply:
performance_evaluation_joey, performance_evaluation_pyTorch and the driver class performance_evaluation.py is the modified and refactor lenet evaluation test that I performed with the correct loss function to understand the  reason for 'Nan' in the forward pass of the lenet